packages/components/src/components/AddressInput/AddressInput.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { AddressInput } from '.';
+import { MockProvider } from 'ethereum-waffle';
+
+describe('AddressInput', () => {
+  it('renders', () => {
+    const provider = new MockProvider();
```

Comment:
This is the [repo](https://github.com/codingwithmanny/web3-react-jest-tests/blob/main/src/__tests__/App.test.tsx) that @codingwithmanny came up with when we were exploring mocking providers.  In the link you'll find how the metamask provider was mocked and you can use that as a guide; however, there doesn't seem to be a universal mock wallet provider as I think everyone kinda has there own implemented standards

Reply:
Thanks for this reference. I'll take a look.

Reply:
Thanks for the review @narbs91!

Reply:
No prob at all, this things looking pretty dope you've all done an awesome job!  Keep it up 🚀

Reply:
> This is the repo that @codingwithmanny came up with when we were exploring mocking providers.

@narbs91 This worked for me and tests are now passing. Thanks for helping me out. 😃